models/public/efficientnet-b0-pytorch/model.yml

```diff
@@ -0,0 +1,72 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  The `efficientnet-b0-pytorch` model is one of the EfficientNet <https://arxiv.org/abs/1905.11946>
+  group of models designed to perform image classification. This model was pretrained
+  in PyTorch\*. All the EfficientNet models have been pretrained on the ImageNet image
+  database. For details about this family of models, check out the repository <https://github.com/rwightman/gen-efficientnet-pytorch>.
+
+  The model input is a blob that consists of a single image 3x224x224 in RGB
+  order. The RGB mean values need to be subtracted as follows: [123.675,116.28,103.53]
+  before passing the image blob into the network. In addition, values must be multiplied
+  by [58.395,57.12,57.375].
+
+  The model output for `efficientnet-b0-pytorch` is the typical object classifier output
+  for the 1000 different classifications matching those in the ImageNet database.
+task_type: classification
+files:
+  - name: model/gen_efficientnet.py
+    size: 40997
+    sha256: 8613d48ca74611d3566d8c02cbf7c92aad6b16a708ffcf21183a7014fecdec09
+    source: https://raw.githubusercontent.com/rwightman/gen-efficientnet-pytorch/a36e2b2cd1bd122a508a6fffeaa7606890f8c882/gen_efficientnet/gen_efficientnet.py
+  - name: model/efficientnet_builder.py
+    size: 18446
+    sha256: 69bb2adc49dc79c8860f36acca910bad6733e23b46ed80551248b90141d5e1b5
+    source: https://raw.githubusercontent.com/rwightman/gen-efficientnet-pytorch/a36e2b2cd1bd122a508a6fffeaa7606890f8c882/gen_efficientnet/efficientnet_builder.py
+  - name: model/helpers.py
+    size: 1097
+    sha256: 0415f198b8a87cb34c9f9aed79f267043010fbd197e27df805bab9d070da82ed
+    source: https://raw.githubusercontent.com/rwightman/gen-efficientnet-pytorch/a36e2b2cd1bd122a508a6fffeaa7606890f8c882/gen_efficientnet/helpers.py
+  - name: model/conv2d_helpers.py
+    size: 6175
+    sha256: e6ba7e878dd28c6d0ccc9707205bf6b58bea664d7fee820d7101a6a8e48990a9
+    source: https://raw.githubusercontent.com/rwightman/gen-efficientnet-pytorch/a36e2b2cd1bd122a508a6fffeaa7606890f8c882/gen_efficientnet/conv2d_helpers.py
+  - name: model/__init__.py
+    size: 32
+    sha256: e0bedda5f5e949b8ace7d4f5cdf80e7e664c0b0a935486b6152078fa61c80c1b
+    source: https://github.com/rwightman/gen-efficientnet-pytorch/raw/a36e2b2cd1bd122a508a6fffeaa7606890f8c882/gen_efficientnet/__init__.py
+  - name: efficientnet-b0.pth
+    size: 21376958
+    sha256: d6904d92f92ccdca67c9717f9d119392d658577f99e5ce021b57d157985783db
+    source: https://github.com/rwightman/pytorch-image-models/releases/download/v0.1-weights/efficientnet_b0-d6904d92.pth
+pytorch_to_onnx:
+  - --model-path=$dl_dir
+  - --model-name=efficientnet_b0
```

Comment:
I noticed you're using the retrained model for B0, but models converted from TensorFlow for B5 and B7. Why is that?

Reply:
Because B0, B1, B2 are pretrained in PyTorch, and for B3-B7 weights are converted from TF.
See [https://github.com/rwightman/gen-efficientnet-pytorch](https://github.com/rwightman/gen-efficientnet-pytorch).

Reply:
But there's also a converted version of B0. Why not use that?

Reply:
Ping?

Reply:
We had an IRL discussion where @eizamaliev made the case that it's preferable to use retrained weights where they are available, since the converted weights cannot be replicated using the PyTorch code alone, so they don't really belong in something called a PyTorch implementation. If retrained weights for B3-B7 models become available too, we might replace the converted weights with them.

I think that is a fair point, so I'm resolving this.